app/src/main/java/com/my/playlistmaker/data/converters/DbConvertors.kt

```diff
@@ -0,0 +1,95 @@
+package com.my.playlistmaker.data.converters
+
+import com.my.playlistmaker.domain.models.Track
+import com.my.playlistmaker.data.db.entity.PlaylistEntity
+import com.my.playlistmaker.data.db.entity.PlaylistTracksEntity
+import com.my.playlistmaker.data.db.entity.TrackEntity
+import com.my.playlistmaker.domain.models.Playlist
+
+class DbConvertors {
+    fun map(track: Track): TrackEntity {
+        return TrackEntity(
+            track.trackName,
+            track.artistName,
+            track.trackTimeMillis,
+            track.artworkUrl100,
+            track.trackId,
+            track.collectionName,
+            track.releaseDate,
+            track.primaryGenreName,
+            track.country,
+            track.previewUrl,
+            addingTime = System.currentTimeMillis()
+        )
+    }
```

Comment:
```suggestion
    fun map(track: Track): TrackEntity = TrackEntity(
            track.trackName,
            track.artistName,
            track.trackTimeMillis,
            track.artworkUrl100,
            track.trackId,
            track.collectionName,
            track.releaseDate,
            track.primaryGenreName,
            track.country,
            track.previewUrl,
            addingTime = System.currentTimeMillis()
        )
```

Для краткости можно писать вот так